api/src/test/java/org/jmisb/api/klv/KlvParserTest.java

```diff
@@ -62,6 +64,49 @@ private void doParse() throws KlvParseException {
         assertTrue(rawMessage.getIdentifiers().isEmpty());
     }
 
+    @Test
+    public void checkParseStream() throws KlvParseException {
+        doParseStream();
+    }
+
+    private void doParseStream() throws KlvParseException {
```

Comment:
This is good, but I'd like to see a test that checks a second message works too. 

https://github.com/WestRidgeSystems/jmisb/blob/2.x/api/src/test/java/org/jmisb/api/klv/KlvParserTest.java#L75 is a model you can copy from.

(Or refactor and re-use, depending on how much time you have - I am happy to do some janitorial work afterwards if that is easier).

Reply:
Updated to push two messages through and check them.